OnlineShopUI/online-shop-ui/src/components/ModalLogin/Login.js

```diff
@@ -70,15 +113,16 @@ export default function Login() {
                   label="Запомнить меня"
                 />
               </FormGroup>
-            </div>
+            </form>
             <div className="login__actions">
               <div className="login__sign">
-                <button
+                <input
+                  type="submit"
                   className="login__sign-btn"
                   onClick={() => setFlag(!flag)}
-                >
-                  Войти
-                </button>
+                  value="Войти"
```

Comment:
Probably it's better to use the English language on UI

Reply:
Changed the language in the entire modal window